WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs

```diff
@@ -182,6 +176,41 @@ protected WalletViewModel(Wallet wallet) : base(wallet)
 
 	public TileLayoutViewModel? CurrentLayout => Layouts?[LayoutIndex];
 
+	private async Task HandleAuthorizationInteractionAsync(InteractionContext<Unit, bool> interaction)
+	{
+		bool authorized = false;
+		if (!string.IsNullOrEmpty(wallet.Kitchen.SaltSoup()))
+		{
+			bool retry;
+			do
+			{
+				var dialog = new PasswordAuthDialogViewModel(wallet);
```

Comment:
@SuperJMN  Error CS0103: The name 'wallet' does not exist in the current context.